### ISSUE TITLE
iOS 13.1 Shared Users Access

### ISSUE THREAD

**Issue by erpeters3** [nfarina/homebridge#2318]
Since updating to 13.1 last beta (hoped it would fix with GM release but it has not) the main user has no issues but all shared users cannot access any HomeBridge devices.  Devices from other hubs work correctly, even for shared users.  Installed latest NPM, HomeBridge, and plugins to no avail.

**Comment by erpeters3** [comment]
Slight change.  Perhaps related.   Eventually my shared users were able to access after I wiped every device iCloud and rebooted the hubs (appleTV). Now however no one can access remotely even though non-HomeBridge devices work perfectly.   All of this worked before 13.1.

**Comment by erpeters3** [comment]
I go back to my initial issue.   After some time all users can no longer control anything on the HoneBridge hub from in the same network.   All other hubs work great.  Anyone else?

**Comment by macjeff** [comment]
All work fine for me.  iOS 13.1 on all devices.  13.0 on Apple TV. 

I did have issues before I updated the Apple TV with users.  You must have either an iPad or Apple TV as the hub I think for shared users to work correctly.   

Jeff

**Comment by erpeters3** [comment]
Perhaps an issue that I have about 10 hubs.   I’ll try updating all the appleTVs and iPads tomorrow.  The HomePods are another issue.

**Comment by macjeff** [comment]
It should go with the newest version but best to update them all.   

Jeff

**Comment by iRayanKhan** [comment]
Honestly I'd wait for the HomePod iOS 13.x GM. Home Sharing has been broken for 2 months for me. Constant phone calls w/ Apple Support and Engineers

**Comment by erpeters3** [comment]
Hopefully that’s it.  I now see that remote access for me as the home owner and all access (remote and local) for shared users is broken.  Perhaps related to the new AppleTV multiple users etc.   I’ll update once 13.1 comes out for HomePod.

**Comment by erpeters3** [comment]
I read elsewhere that removing AppleTVs from the hub list and relying on HomePod on 12.4 should fix.  I disabled hub in all ATVs and no change.  Family is about to mutiny on having to get up and turn lights off by hand.  Sigh.

**Comment by mafyata** [comment]
I don’t have such a problem. I am using ipad and iphone with ios13.1 , atv 4k (as hub) with tvos 13. Other people that I shared home with them are using iphones with ios 12.4.1

**Comment by erpeters3** [comment]
This would lead me to believe perhaps it is the HomePods that cannot be disabled. I might consider removing all of them but that’ll take a while.   Very frustrating but will keep digging.  Are there any logs anywhere that might show me why the shared user devices think devices are unavailable?

**Comment by iRayanKhan** [comment]
When I was looking trough logs on the guests devices (Settings > Privacy > Analytics) I saw “homekit_sync_limit” been on the phone with apple for nearly 2 months on this issue. They’re doing their best.

**Comment by erpeters3** [comment]
I disabled every HomePod and went back to a single AppleTV.  Reset HomeBridge persists and accessories, rebuilt from scratch.  Still does not work.  Unless someone here finds some great idea, I guess I am stuck waiting on HomePod 13.1 and perhaps even 13.2.  Really frustrating mess Apple has created this year with iOS 13.

**Comment by iRayanKhan** [comment]
Try making a new HomeKit home?

**Comment by erpeters3** [comment]
Probably the one thing i haven’t done.   I’ll give that a go when i get home.

**Comment by iRayanKhan** [comment]
Best of luck, I have 70+  Accessories and configuring it all would suck, might try to add fake accessories and see if that works.

**Comment by erpeters3** [comment]
A new home kit home, a total rebuild, and 3 hours of my life solved it.  Thanks iRayanKhan.

**Comment by iRayanKhan** [comment]
@erpeters3 No way, about to try that.

**Comment by erpeters3** [comment]
Yep.  Recommend you add one hub first.  See if it fixes it.   Did for me.  Frankly haven’t rebuilt gone from scratch since it launched so probably overdue.   A lot of things are just working smoother for now.

**Comment by iRayanKhan** [comment]
I’m curious, what version are you’re hubs on, how many of each do you have, and which did you add first. Sorry I’m asking all this since it’s a large set-up for me.

**Comment by erpeters3** [comment]
Everything is latest as part of this troubleshooting.     I reenabked all of my ATVs and HonePods.  Then created a new home.   In homebridge I change the user hex code thing and the HomeKit code.  Just in case.  Then started the new home.  Then added homebridge after removing from original home.  Tested remote access by dropping WiFi from phone and accessing over LTE. ONCE THAT WORKED I MOVED EVERYTHING ELSE LIKE hue Lutron, etc.

**Comment by iRayanKhan** [comment]
Congrats. I'm gonna try this tomorrow, Plugin by Plugin. I hope it works and doesn't just break again :P. I did hear that the HomeKit lock I use is broken in iOS 13. So that isn't gonna be fun to deal with.

**Comment by nicklayb** [comment]
Hey guys, sorry to bump an old thread. I'm having the exact same issue. I have an Homebridge server, an Aqara bridge and s Hue bridge. I have 2 ATVs as Hubs and one iPad (the iPad's not mine so I'm not using it as a Hub).

The issue is that I see all of my accessories on my iPhone but my girlfriend's iPhone and iPad are all missing Homebridge's accessories (even the bridge is missing in home setting). Aqara's and Hue's devices all works perfectly. Also worth noting, I removed her from family sharing and add her back. Accessories were there for like an hour then magically disappeared.

Does re-creating a home lasted for you guys? I don't really want to do it if I'm about to do it again in two weeks.

Thank you!

**Comment by iRayanKhan** [comment]
I made a whole new HomeKit home when iOS 13 came out. It has fixed most of the issues I've had, especially shared access. If you run into more issues, you should open up a new issue.
